components/Theme.tsx

```diff
@@ -0,0 +1,15 @@
+import { createMuiTheme, Theme, PaletteType } from '@material-ui/core'
```

Comment:
Just noticed this has been renamed from `/components/theme.tsx` to `/components/Theme.tsx` (capitalised). TBH if it's going to be renamed, perhaps `/theme.tsx` is best, since it isn't a component? If not, just revert to the old name and I'll consider moving it later.